zk_toolbox/crates/zk_inception/src/commands/chain/mod.rs

```diff
@@ -35,6 +36,9 @@ pub enum ChainCommands {
     /// Deploy paymaster smart contract
     #[command(alias = "paymaster")]
     DeployPaymaster(ForgeScriptArgs),
+    /// Update Token Multiplier Setter address on L1
+    #[command(alias = "updateTokenMultiplierSetter")]
```

Comment:
you don't need to specify an alias i n this way

Reply:
What's the best way to do it then?

Reply:
makes sense - removed